debian/cloud-init-base.config

```
@@ -52,28 +64,31 @@ if [ -f "${old_dpkg_cfg}" -a ! -f "$dpkg_cfg" ]; then
     dpkg_cfg="${old_dpkg_cfg}"
    echo "WARN: reading value from ${old_dpkg_cfg}" 1>&2
 fi
+
+migrate_debconf_to_cloud_init_base datasources
+
 if [ -f "$dpkg_cfg" ]; then
     if get_yaml_list "$dpkg_cfg" datasource_list NOTFOUND &&
         val="$RET" && [ "$val" != "NOTFOUND" ]; then
-        db_set cloud-init/datasources $val
+        db_set cloud-init-base/datasources $val
```

Quote this value just in case for var expansion
```suggestion
        db_set cloud-init-base/datasources "$val"
```

---

Fixed, thanks.